dist/common/scripts/scylla_raid_setup

```diff
@@ -14,10 +14,84 @@ import grp
 import sys
 import stat
 import distro
+import logging
+import pyudev
 from pathlib import Path
 from scylla_util import *
 from subprocess import run, SubprocessError
```

Comment:
Can you provide more information about the problem? Why isn't /dev/disk/by-uuid available?

Reply:
@syuu1228  ?

Reply:
@avikivity See https://github.com/scylladb/scylla-enterprise/issues/2835, at least on this environment we hitting issue to get /dev/disk/by-uuid, and wipefs doesn't fix the problem.
Also previous scylla_raid_setup issues can be similar with https://github.com/scylladb/scylla-enterprise/issues/2835, since we closed these issues after we added `udevadm settle` but we don't really know `udevadm settle` fixed the problem or not.

I want to avoid breaking setup even such problem exists on the environment.
With this patch, setup will continue with warning message.

Reply:
@avikivity ping

Reply:
Hmm I don't understand it. Maybe the problem is that udevadm settle commits the changes to /dev, but systemctl hasn't picked them up yet.

Reply:
Maybe systemctl daemon-reload will pick up changes to /dev.

Is it easy to reproduce the problem? I think that's better than the big chunk of code below.

Reply:
Please also ask on the mailing list if there is a way to wait for systemd to synchronize with /dev updates.

Reply:
> Hmm I don't understand it. Maybe the problem is that udevadm settle commits the changes to /dev, but systemd hasn't picked them up yet.

No, it's VMware not adding UUID to devices, unless instructed to do so.

Reply:
Ah. That's unfortunate.

Reply:
> > Hmm I don't understand it. Maybe the problem is that udevadm settle commits the changes to /dev, but systemd hasn't picked them up yet.
> 
> No, it's VMware not adding UUID to devices, unless instructed to do so.

@mykaul I do not have enough knowledge about VMware environment, but are we sure it's because VMware?

Seems like VMware does not provide /dev/disk/by-id unless we specify `disk.EnableUUID = "TRUE"` on it's configuration:
https://www.suse.com/de-de/support/kb/doc/?id=000016951

But it's not /dev/disk/by-uuid, it can be different problem.
by-uuid is filesystem's UUID not devices, so I think it's not related with the (virtual) device.

@avikivity We can add it to the changelog but I'm not really sure it's related with VMware or not.

Reply:
To the best of my knowledge, the issue is with VMware. The UUID is certainly from the HW (you can try with a KVM VM and libvirt if you wish to see it)

Reply:
It's probably vmware - real disks have uuid, and I guess vmware omits the uuids for repeatability.

Reply:
See this for example: https://sort.veritas.com/public/documents/sfha/6.2/vmwareesx/productguides/html/sfhas_virtualization/ch10s05s01.htm

likely the default is off.

Reply:
Okay, updated the commitlog.